resources/css/filament/admin/theme.css

```
@@ -200,3 +200,23 @@
     border: none;
     border-radius: 50%;
 }
+
+.fi-page-sub-navigation-sidebar-ctn {
+    @apply bg-white border border-gray-950/5 rounded-xl dark:bg-gray-900 dark:border-white/10;
+}
+
+.fi-page-sub-navigation-sidebar {
+    @apply p-4;
```

I think this should probably be a touch less

```suggestion
    @apply p-3;
```